pypowsybl/glsk.py

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright (c) 2022, RTE (http://www.rte-france.com)
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+from typing import List as _List
+from datetime import datetime
+import pypowsybl._pypowsybl as _pypowsybl
+from pypowsybl.network import Network as _Network
+
+#GLSKImporter helper class
+class GLSKImporter:
```

Comment:
From the API point of view: 
for better consistency with network API, maybe we could have a `GLSKDocument` class and a `load` method ?

```python
gslk_document = pp.glsk.load('/path/to/doc')
gslk_document.get_glsk_factors(...)
```

In the current API, it feels like the document will be imported after the creation of the importer, whereas it is actually imported as soon as the object is built.

Also: we should support `os.PathLike` as an input to the loading method, together with `str` (see network.load).